e2e/cypress/integration/enterprise/ldap_group/channel_modes_spec.js

```diff
@@ -73,10 +73,16 @@ describe('LDAP Group Sync - Test channel public/private toggle', () => {
         cy.getCurrentChannelId().then((id) => {
             cy.visit(`/admin_console/user_management/channels/${id}`);
             cy.get('#channel_profile').contains('Town Square');
-            cy.get('#channel_manage .group-teams-and-channels--body').find('button').eq(0).should('have.class', 'false');
-            cy.get('#channel_manage .group-teams-and-channels--body').find('button').eq(1).contains('Public');
-            cy.get('#channel_manage .group-teams-and-channels--body').find('button').eq(0).should('have.class', 'disabled');
-            cy.get('#channel_manage .group-teams-and-channels--body').find('button').eq(1).should('have.class', 'disabled');
+            cy.get('#channel_manage').scrollIntoView().should('be.visible').within(() => {
+                cy.get('.line-switch').first().within(() => {
+                    cy.findByText('Sync Group Members').should('be.visible');
+                    cy.get('#syncGroupSwitch').should('be.disabled');
```

Comment:
not sure if I'm missing something but this `get` is failing for me, and I don't see a normal ID with that name in the DOM. there are however a couple `data-testid`s there.

I had to change this to `cy.findByTestId('syncGroupSwitch-button')` to pass

Reply:
Thanks Cass! Not sure why but it works for me. Anyway, I changed per your suggestions and it passed on me as well. Please take a look again.